build/compile_descriptor

```diff
@@ -20,20 +20,26 @@ is_up_to_date () {
 
 set -e
 
+if [ -d "./ebin" ]; then
+    OUT="./ebin"
+else
+    OUT="$REBAR_DEPS_DIR/gpb/ebin"
+fi
+
```

Comment:
Nah, it is probably just me mixing building with make every now and then, mostly because of old habits. Let's drop this use-case.